client/src/styles/bundle.scss

```diff
@@ -2,10 +2,16 @@
 @import "variables";
 
 // Thirdparty
+@import "~bootstrap/scss/functions";
```

Comment:
Were the functions and variables required for the existing SCSS to compile?

Reply:
It was before I stripped out the edit and delete button. Will re-check if this is still needed.

Reply:
Yup, required indeed.